qiime/metadata.py

```diff
@@ -14,12 +16,30 @@ def __init__(self, dataframe):
     def __repr__(self):
         return self._dataframe.to_json()
 
+    @staticmethod
+    def load(path):
+        try:
+            df = pd.read_csv(path, sep='\t', index_col=0, dtype=object)
```

Comment:
Thanks @wasade. 

I think the only concerns, which might not be are:
- values with leading/following empty spaces
- blank rows or columns
- if the first row is gonna be used like the index that means that it can have any label and this might completely confuse users. However, not hard feelings about this one.
- in past versions of pandas if you index on read, it will try to auto cast the values of the index (even ignoring the dtype parameter), not sure if this is still the case with the version that skbio uses